docs/Rest-API-description.md

```diff
@@ -262,21 +266,21 @@ Response body on success:
 }
 ```
 
-| Element        | Type    | Description                                                       |
-|----------------|---------|-------------------------------------------------------------------|
-| face.image_id  | UUID    | UUID of the face                                                  |
-| fase.subject   | string  | <subject> of the person, whose picture was saved for this api key |
-| page_number    | integer | page number                                                       |
-| page_size      | integer | **requested** page size                                           |
-| total_pages    | integer | total pages                                                       |
-| total_elements | integer | total faces                                                       |
+| Element        | Type    | Description                                                                           |
+|----------------|---------|---------------------------------------------------------------------------------------|
+| face.image_id  | UUID    | UUID of the face                                                                      |
+| fase.subject   | string  | <subject> of the person, whose picture was saved in selected subject for this api key |
```

Comment:
Please remove `in selected subject` as it could be used for all subjects if `subject` is not defined
also, please fix the typo - `fase` to `face`

Reply:
Did this